.jenkinsci-new/builders/x64-linux-build-steps.groovy

```diff
@@ -61,19 +61,22 @@ def buildSteps(int parallelism, List compilerVersions, String build_type, boolea
     if (sanitize){
       cmakeOptions += " -DSANITIZE='address;leak' "
     }
-    sh "docker network create ${env.IROHA_NETWORK}"
+
+    sh "docker network create ${env.IROHA_NETWORK} || true"
```

Comment:
Shouldn't we fail the build if something goes wrong: either creating Docker network of running a Docker container? Why do you hide non-zero exit codes with ` || true`?